CHANGES.rst

```diff
@@ -1,4 +1,4 @@
-0.14.0 (Unreleased)
+0.14.0 (2019-10-27)
```

Comment:
Any particular reason you chose the 27th, as opposed to today's date (10-25)?

Reply:
Really, I'm 2 days ahead. will fix

Reply:
You're always way ahead of the rest of us ...